### ISSUE TITLE
Convergence Analysis for Training Stochastic Neural Networks via Stochastic Gradient Descent

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#370]
### 論文リンク
https://arxiv.org/abs/2212.08924

## 概要
本論文では，確率的ニューラルネットワーク(SNN)を学習するための新しいサンプルワイズバックプロパゲーション法の収束を証明するための数値解析を行う．SNNの構造は、確率微分方程式(SDE)の離散化として定式化される。確率的最適制御の枠組みを導入し、確率的最適制御ソルバーの効率を向上させるために、随伴後退SDEに対するサンプルワイズ近似スキームを適用し、SNNの学習のためのバックプロパゲーションと等価であることを示す。SNNパラメータの最適化について、凸性の仮定がある場合とない場合の収束解析を導出した。特に、凸最適化の場合、SNNの学習ステップ数は層数の二乗に比例する必要があることが解析から示された。解析結果を検証するために数値実験を行い、サンプルワイズバックプロパゲーション法によるSNNの学習性能を機械学習のベンチマーク例により検証する。
